integration/benchmark/scripts/addKeyToAccountTransaction.cdc

```diff
@@ -7,7 +7,7 @@ transaction(keys: [[UInt8]]) {
       )
       signer.keys.add(
         publicKey: publicKey,
-        hashAlgorithm: HashAlgorithm.SHA3_256,
+        hashAlgorithm: HashAlgorithm.SHA2_256,
```

Comment:
why the regression to an older hashing algo?

Reply:
this is the algorithm used by the service account key. It's easier to just use the same key everywhere so we don't need to track state between runs